### PR TITLE
Issue 5507 - Fix 1.4.3 GitHub PR CI

### PR DESCRIPTION
**Pull request by droideck**
**Description**
Our CI fails in many places because of the missing backports and Fedora package changes.

Backport the missing peaces and fix CI images.
Update package-lock.json
Add Requires: acl.
Remove failing perl_enable invocation.
Don't set max age if changelog doesn't exist.
Fix dscreate, ssl_test, ssl_version, healcheck, fourwaymmr tests

Fixes: https://github.com/389ds/389-ds-base/issues/5339
Fixes: https://github.com/389ds/389-ds-base/issues/5507

Reviewed by: ?

### CODE REVIEW

File: dirsrvtests/tests/suites/healthcheck/healthcheck_test.py [hunk 1]
```diff
@@ -65,11 +58,10 @@ def run_healthcheck_and_flush_log(topology, instance, searched_code=None, json=F
...
 
 def set_changelog_trimming(instance):
-    log.info('Get the changelog enteries')
-    inst_changelog = Changelog5(instance)
-
     log.info('Set nsslapd-changelogmaxage to 30d')
-    inst_changelog.add('nsslapd-changelogmaxage', '30')
+
+    cl = Changelog5(instance)
+    cl.replace('nsslapd-changelogmaxage', '30')
```
Comment (mreynolds389) [inline dirsrvtests/tests/suites/healthcheck/healthcheck_test.py]: I think this is supposed to be "30d"

**Comment by mreynolds389** [comment]
@droideck I think you're missing changes to fedse.c and template-dse.ldif.in.  I think there should be the new pwdchan password plugins listed in these two files.

**Comment by droideck** [comment]
> @droideck I think you're missing changes to fedse.c and template-dse.ldif.in. I think there should be the new pwdchan password plugins listed in these two files.

I've reverted the "PBKDF2-SHA512 by default" change for now because it requires a massive Rust backport.

One thing is not fully clear to me. Do we want to make Rust in 1.4.3 mandatory?
If we add PBKDF2-SHA512 functionality with this [PR](https://github.com/389ds/389-ds-base/pull/4457/files), we'll have to always build with Rust enabled. So I want to clarify if it's our intention...

**Comment by mreynolds389** [comment]
> > @droideck I think you're missing changes to fedse.c and template-dse.ldif.in. I think there should be the new pwdchan password plugins listed in these two files.
> 
> I've reverted the "PBKDF2-SHA512 by default" change for now because it requires a massive Rust backport.
> 
> One thing is not fully clear to me. Do we want to make Rust in 1.4.3 mandatory? If we add PBKDF2-SHA512 functionality with this [PR](https://github.com/389ds/389-ds-base/pull/4457/files), we'll have to always build with Rust enabled. So I want to clarify if it's our intention...

1.4.3. is only used for RHEL builds now, so it is fine if it's RUST by default.  I thought this is what you were doing actually.  If it's still a lot of work then it can be skipped for now.

File: dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py [hunk 2]
```diff
@@ -439,13 +450,16 @@ def test_replica_backup_and_restore(topo_m4):
         5. Should success
     """
     # Testing bug #830335: Taking a replica backup and Restore on M1 after deleting few entries from M1 nad M2
+    # Make sure to update S1 to avoid useless replication delay
```
Comment (progier389) [inline dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py:453]: I would not change the supplier in that test. : We do not have lot of 4 supplier tests and IMHO it is a good idea to have a test in which the changes are not issued on supplier 1 .
IMHO we should rather keep creating the user on supplier 3 but wait 
repl.wait_for_replication(topo_m4.ms["supplier3"], topo_m4.ms["supplier1"]) before waiting for supplier1 -> supplier2

Comment (droideck) [reply dirsrvtests/tests/suites/fourwaymmr/fourwaymmr_test.py:453]: Closing the conversation for now, as we've discussed in the meeting that this part of the PR is a pure backport from the main. And we can change it through the usual PR process main->backports in the future.
And probably, we should, as fourwaymmr test suite still has its issues and can fail from time to time.

**Review by progier389 — APPROVED** [review]
Better have Viktor's feeling about the .github subtree changes,
But with the clarification we got in the meeting, that is OK on my side.

**Review by vashirov — APPROVED** [review]
I'm happy with the changes, thanks for syncing GH actions workflows!

**Pull request merged**